EIPS/eip-1319.md

```diff
@@ -166,6 +166,62 @@ release versions while allowing them to use any versioning schema they choose.
 
 Registries may offer more complex `read` APIs that manage requests for packages within a semver range or at `latest` etc. This EIP is agnostic about how tooling or registries might implement these. It recommends that registries implement [EIP 165](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md) and avail themselves of resources to publish more complex interfaces such as [EIP 926](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-926.md).
 
+## EthPM URIs
+
+EthPM URIs are used to identify on-chain erc1319-compliant (ethPM) registries and their packages. The goal of this URI specification is to provide a human and machine readable string that can uniquely identify any package release across any registry. This simple datatype allows for quick, one-click installs and convenient transportion of ethpm packages between tooling/frameworks.
+
+### Format
+
+`scheme://registry_address:chain_id/package_name@package_version/json_pointer`
+
+#### `scheme`
+- Required
+- Must be one of `ethpm` or `erc1319`
+
+#### `registry_address`
+- Required
+- This **should** be an ENS name whenever possible, or a `0x`-prefixed, checksummed address. 
```

Comment:
This reads as if an `ENS` name is preferable to the other formats.  Maybe reword as 

> This **should** be either an ENS name or a `0x`-prefixed, checksummed address.   ENS names are more suitable for cases where mutability of the underlying asset is acceptable and there is implicit trust in the owner of the name.  `0x` prefixed addresses are more preferable in higher security cases to avoid needing to trust the controller of the name.